### ISSUE TITLE
Issue with 0's in error values

### ISSUE THREAD

**Issue by ryanjmccall** [dwkim78/upsilon#4]
If we pass an array for errors that contains any 0 values, then there are issue due to the weight calculation `self.weight = 1. / self.err`.  (https://github.com/dwkim78/upsilon/blob/master/upsilon/extract_features/extract_features.py#L90). Later the random forest classifier bombs b/c it asserts_all_finite. I see that the API basically says, if no errors are provided, then upsilon will calculate errors based on std. However, real data will come in with 0 for the error and even one occurrence can ruin everything it seems. Maybe an easy solution would be to `assert self.err.all()` in  the constructor, or even better, to fill in the std for all 0 error values.

**Comment by ryanjmccall** [comment]
how about this?

```            
if not self.err.all():
     np.putmask(self.err, self.err == 0, np.std(self.mag))
```

**Comment by dwkim78** [comment]
Hi Ryan, 

Thanks for reporting the issue. It'll be fixed at the next release (v 1.2.7).
